nixos/modules/image/repart-verity-store.nix

```diff
@@ -0,0 +1,198 @@
+# opinionated module that can be used to build nixos images with
+# a dm-verity protected nix store and a volatile rootfs
```

Comment:
```suggestion
# a dm-verity protected nix store
```
This module does not actually handle the rootfs, which this, at least to me, read as